### DISCUSSION TITLE
Is it worth the headache of trying to make splashscreens work on iOS?

### DISCUSSION THREAD

**Discussion by newsonaut** [aeharding/voyager#1249] [Q&A]
I’ve been having trouble getting splashscreens to work on my iPad and iPhone SE for a personal PWA that I’m working on. I’m an admirer of Voyager, so I thought I’d have a look at your source code to see your approach.

Looks like you have dozens of links for apple-touch-startup-image, yet not one of them is working for Voyager on my iPad or iPhone. 

I haven’t submitted this as an issue, because I’m sure you have better things to do, but I was wondering if you have any insights on the iOS splashscreen issue.

Is it worth pursuing?

**Comment by aeharding** [comment]
Hmm, I'm sure. Voyager uses https://github.com/elegantapp/pwa-asset-generator to generate splashes. It seems to be working fine on my iPhone.

**Reply by newsonaut** [reply]
May I ask which model iPhone you're using?

**Reply by aeharding** [reply]
Iphone 13 mini

**Reply by newsonaut** [reply]
Thanks. I'll keep trying.
